home/service/mail/docker-compose.yml

```diff
@@ -1,10 +1,14 @@
 services:
   mailhog:
-    image: mailhog/mailhog:v1.0.1
+    image: axllent/mailpit
```

Comment:
Better to pin to a tag (and ideally sha256 hash) than use latest

Reply:
Hi Kieren, long time no see :)
Sure, I have pinned it to latest stable version.